literacywidget.js

```diff
@@ -62,17 +63,18 @@ class LiteracyWidget extends Widget {
 
         this.input = this.element.querySelector( `input[name="${name}"]` );
 
+        console.log("derp");
```

Comment:
👍  This was helpful for testing, good reminder to restart `npx grunt develop` after pulling this branch.

Suggestion:

```suggestion
```